tests/backends/mysql/tests.py

```diff
@@ -89,3 +91,24 @@ def test_isolation_level_validation(self):
         )
         with self.assertRaisesMessage(ImproperlyConfigured, msg):
             new_connection.cursor()
+
+
+@unittest.skipUnless(connection.vendor == 'mysql', 'MySQL tests')
+class CallProcTest(BaseCallProcTest, unittest.TestCase):
+    procedure_without_params_sql = '''
```

Comment:
Another possibility could be to consider these test strings "database features" and put them there.